plugins/qeta-backend/src/service/router.ts

```diff
@@ -38,6 +48,14 @@ export interface RouterOptions {
   permissions?: PermissionEvaluator;
 }
 
+const DEFAULT_IMAGE_SIZE_LIMIT = 2500000;
+const SUPPORTED_FILES_TYPES = [
```

Comment:
These are just to have a default value in case whoever uses the plugin doesn't fill in the app-config. To ease the use of the plugin for those who are not familiar with it.